pkg/register/websocket.go

```diff
@@ -32,6 +32,7 @@ const (
 	MsgUndefined MessageType = iota
 	MsgReady
 	MsgSmbios
+	MsgSystemData
 	MsgLabels
 	MsgGet
 	MsgLast = MsgGet // MsgLast must point to the last message
```

Comment:
The new message should be put just before MsgLast (and MsgLast should be updated to MsgSystemData).
This way we preserve old messages.
Something like:

```
...
MsgSmbios
MsgLabels
MsgGet
MsgSystemData
MsgLast = MsgSystemData // MsgLast must point to the last message

```

Reply:
thanks!